docs/rules/no-restricted-paths.md

```diff
@@ -9,71 +9,86 @@ In order to prevent such scenarios this rule allows you to define restricted zon
 
 This rule has one option. The option is an object containing the definition of all restricted `zones` and the optional `basePath` which is used to resolve relative paths within.
 The default value for `basePath` is the current working directory.
-Each zone consists of the `target` path and a `from` path. The `target` is the path where the restricted imports should be applied. The `from` path defines the folder that is not allowed to be used in an import. An optional `except` may be defined for a zone, allowing exception paths that would otherwise violate the related `from`. Note that `except` is relative to `from` and cannot backtrack to a parent directory.
+Each zone consists of the `target` path and a `from` path. The `target` is the path where the restricted imports should be applied. The `from` path defines the folder that is not allowed to be used in an import. An optional `except` may be defined for a zone, allowing exception paths or glob patterns that would otherwise violate the related `from`. Note that `except` is relative to `from` and cannot backtrack to a parent directory.
 
-### Examples
+## Examples
+
+### Basic example
 
 Given the following folder structure:
 
 ```
 my-project
 ├── client
-│   └── foo.js
-│   └── baz.js
+│   └── homepage.js
+│   └── about.js
 └── server
-    └── bar.js
+    └── api.js
 ```
 
-and the current file being linted is `my-project/client/foo.js`.
+and the current file being linted is `my-project/client/homepage.js`.
 
-The following patterns are considered problems when configuration set to `{ "zones": [ { "target": "./client", "from": "./server" } ] }`:
+and the configuration set to:
 
 ```js
-import bar from '../server/bar';
+{ "zones": [ {
+    "target": "./client",
+    "from": "./server"
+} ] }
 ```
 
-The following patterns are not considered problems when configuration set to `{ "zones": [ { "target": "./client", "from": "./server" } ] }`:
```

Comment:
The "The following patterns" wording follows eslint's own docs; please restore it in both places.